lib/DataRepeater.js

```diff
@@ -615,6 +615,7 @@ var DataRepeater = module.exports = kind(
 			}
 			else {
 				model[p] = select;
+				if(c) c.syncBindings({force: true, all: true});
```

Comment:
Is this really the only way to do this? Won't it automatically sync when not using POJO's? So doing this blindly with `force: true` seems redundant and wasteful.